News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/helper/Search.java

```diff
@@ -48,6 +51,8 @@ private static String getFeedSQLStatement(final long idFeed,
             sql = dbConn.getAllItemsIdsForFeedSQLFilteredByTitle(idFeed, false, false, sortDirection, searchString);
```

Comment:
I think in line 49 we should set the default to both now, what do you think?

Reply:
Needs to be adjusted in a few places actually: 

- 1: https://github.com/nextcloud/news-android/blob/e36385b93b27ebe8cb160e25a12310b39fa70c8b/News-Android-App/src/main/res/xml/pref_general.xml#L118
- 2: https://github.com/nextcloud/news-android/blob/e36385b93b27ebe8cb160e25a12310b39fa70c8b/News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/helper/Search.java#L46
- 3: https://github.com/nextcloud/news-android/blob/e36385b93b27ebe8cb160e25a12310b39fa70c8b/News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/helper/Search.java#L61